src/functions.js

```diff
@@ -77,8 +77,6 @@ const decryptDataKey = metadata => new Connector(metadata.masterKeyAlias)
         Promise.reject(new Error('could not decrypt data key from any region'))));
   });
 
-const base64EncodeUint8Array = arr => Buffer.from(arr).toString('base64');
-
```

Comment:
reverting this and moving into connector so the backwards compatibility tweaks are in the same place